### PR TITLE
Improve PCH by adding more commonly used headers

### PR DESCRIPTION
**Pull request by vittorioromeo**
Adding these headers results in a full-rebuild speedup of 150-250ms on my machine.

### CODE REVIEW

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

File: src/SFML/System/Win32/WindowsHeader.hpp [hunk 1]
```diff
@@ -44,4 +44,12 @@
 #define WINVER 0x0501
 #endif
 
+#ifndef UNICODE
+#define UNICODE  1
```
Comment (eXpl0it3r) [inline src/SFML/System/Win32/WindowsHeader.hpp:48]: There should only be one space

Comment (vittorioromeo) [reply src/SFML/System/Win32/WindowsHeader.hpp]: Should be fixed now, force-pushed.

**Comment by vittorioromeo** [comment]
SFMLBuildMaster: Build this please.

**Review by ChrisThrasher — COMMENTED** [review]
As someone who can't use PCH (no macOS support) I'm mostly indifferent. If the rest of the team likes it then it's fine with me.

**Pull request merged**